frontend/public/components/pod.tsx

```diff
@@ -379,14 +380,14 @@ const PodTableRow = connect<PodTableRowPropsFromState, null, PodTableRowProps>(p
           columns={columns}
           columnID={podColumnInfo.memory.id}
         >
-          {bytes ? `${formatBytesAsMiB(bytes)} MiB` : '-'}
+          {bytes ? t('public~{{numBytes}} MiB', { numBytes: formatBytesAsMiB(bytes) }) : '-'}
```

Comment:
I'm not sure if units should be translated. Maybe just `cores`? But we should check what we do in other contexts.

Reply:
We're doing the whole cores string elsewhere (namespace.jsx). Looks like bytes units aren't translated. I'll update that one.